app/javascript/backend_lookup.js

```diff
@@ -30,8 +30,10 @@ Blacklight.onLoad(function(){
     }
 
     function updateText(count, el){
-      countText = $("<span>...finds " + count + " results</span>");
-      $(el).after(countText);
+      if (!$(el)[0].nextElementSibling){
```

Comment:
Would you mind adding a comment here explaining what this is doing?

Reply:
Done.